content/applications/productivity/studio/views.rst

```diff
@@ -0,0 +1,301 @@
+  a :guilabel:`Measure Field`.
+- To choose which time interval is used by default to group results, select :guilabel:`Day`,
+  :guilabel:`Week`, :guilabel:`Month`, or :guilabel:`Year` under :guilabel:`Interval`.
+- To change the cohort :guilabel:`Mode`, select either :guilabel:`Retention` (i.e., the percentage
+  of records   staying over a period of time, it starts at 100% and decreases with time) or
+  :guilabel:`Churn` (i.e., the percentage of records moving out over a period of time - it starts at
+  0% and increases with time).
+- To change the way the :guilabel:`Timeline` (i.e., the columns) progresses, select either
+  :guilabel:`Forward’ (from 0 to +15) or :guilabel:`Backward` (from -15 to 0). For most purposes,
+  the :guilabel:`Forward` timeline is used.
+
+.. _timeline/gantt:
+
+Gantt
+-----
+
+The :guilabel:`Gantt` view is used to forecast and examine the overall progress of records. Records
+are represented by a bar under a time scale.
+
+- To prevent users from creating or editing records, untick :guilabel:`Can Create` or :guilabel:`Can
+  Edit`.
+- To fill cells in gray whenever a record shouldn’t be created there (e.g., on weekends for
+  employees), tick :guilabel:`Display Unavailability`.
+
+  .. note::
+     The underlying model must support this feature, and support for it cannot be added using
+     Studio. It is supported for the Project, Time Off, Planning and Manufacturing apps.
+
+- To show a total row at the bottom, tick :guilabel:`Display Total row`.
+- To collapse multiple records in a single row, tick :guilabel:`Collapse First Level`.
+- To choose which way records are grouped by default on rows (e.g., per employee or project), select
+  a field under :guilabel:`Default Group by`.
+- To define a default time scale to view records, select :guilabel:`Day`, :guilabel:`Week`,
+  :guilabel:`Month`, or :guilabel:`Year` under :guilabel:`Default Scale`.
+- To color records on the view, select a field under :guilabel:`Color`. All the records sharing the
+  same value for that field are displayed using the same color.
+
+  .. note::
+     As the number of colors is limited, the same color can end up being assigned to different
+     values.
+
+- To specify with which degree of precision each time scale should be divided by, select
+  :guilabel:`Quarter Hour`, :guilabel:`Half Hour`, or :guilabel:`Hour` under :guilabel:`Day
+  Precision`, :guilabel:`Half Day` or :guilabel:`Day` under :guilabel:`Week Precision`, and
+  :guilabel:`Month Precision`.
+
+.. _views/reporting:
+
+Reporting views
+===============
+
+.. note::
+   The settings described below are found under the view’s :guilabel:`View` tab unless specified
+   otherwise.
+
+.. _reporting/pivot:
+
+Pivot
+-----
+
+The :guilabel:`Pivot` view is used to explore and analyze the data contained in records in an
+interactive manner. It is especially useful to aggregate numeric data, create categories, and drill
```

Comment:
```suggestion
interactive manner. It is specially useful to aggregate numeric data, create categories, and drill
```

Reply:
Especially is the right word here. No change done.